.ibm/pipelines/value_files/values_showcase.yaml

```diff
@@ -213,6 +213,9 @@ global:
         disabled: false
       - package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-global-floating-action-button
         disabled: false
+      # Enable Security Insights plugin
```

Comment:
nit: Please remove this comment. It's clear that it's enabling security insights plugin based on the name of the imported plugin name